skills/debug/SKILL.md

```
@@ -375,10 +375,10 @@ triage → investigate → rca → design → implement → review → synthesiz
 
 ## State Management
 
-Initialize debug workflow using `mcp__exarchos__exarchos_workflow_init`:
+Initialize debug workflow using `mcp__exarchos__exarchos_workflow` with `action: "init"`:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)